### PR TITLE
feat: Aave v2 cap adapters for stables

### PR DESCRIPTION
**Pull request by rustboyar**
Price cap adapters for stablecoins on Aave V2.

### CODE REVIEW

File: tests/AaveV2/EthereumV2Test.t.sol [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+//       AaveV2EthereumAssets.AMPL_ORACLE,
+//       ForkParams({network: 'mainnet', blockNumber: 19620358}),
+//       AdapterParams({
+//         preRequisiteAdapters: preRequisiteAdapters,
+//         adapterCode: AdaptersEthBasedEthereum.AMPLtoETHAdapterCode()
+//       })
+//     )
+//   {}
+// }
+
```
Comment (kyzia551) [inline tests/AaveV2/EthereumV2Test.t.sol:187]:
```suggestion
```

**Review by kyzia551 — APPROVED** [review]
Don't forget to put about DPI on readme

File: scripts/AaveV2/DeployEthereum.s.sol [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+        abi.encode(AaveV3EthereumAssets.WETH_ORACLE, DPI_ORACLE, 18, 'DPI / USD / ETH')
+      );
+  }
+}
+
+contract DeployEthereumAdaptersAndPayload {
+  function _deploy() internal returns (address) {
+    AaveV2EthereumPayload.Adapters memory adapters;
+
+    adapters.usdtAdapter = GovV3Helpers.deployDeterministic(
```
Comment (sakulstra) [inline scripts/AaveV2/DeployEthereum.s.sol:243]: It's fine, but a bit weird to use `GovV3Helpers.deployDeterministic` here - it's just a wrapper around create2utils with a static salt.

**Review by sakulstra — CHANGES_REQUESTED** [review]
looks good to me, but please include a diff snapshot on the tests.
Last time we only noticed a misconfiguration **after** deployment, because on diff-snapshot the price changed unintendedly. Better this time we check before deployment.

**Review by sakulstra — APPROVED** [review]

**Pull request merged**